src/modules/actions.js

```diff
@@ -3,78 +3,92 @@ import Todos from './todos.js';
-      p.id = `${todo.index}`;
-
-      const check = document.createElement('input');
-      check.type = 'checkbox';
-      check.classList.add('todo');
-      check.setAttribute('aria-label', 'Mark as done');
-
-      const taskDescription = document.createElement('span');
-      taskDescription.classList.add('editable');
-      taskDescription.innerText = `${todo.description}`;
-      taskDescription.setAttribute('contenteditable', 'true');
-
-      const rmBtn = document.createElement('button');
-      rmBtn.classList.add('rm-btn');
-      rmBtn.classList.add('hidden');
-      rmBtn.setAttribute('aria-label', 'Delete task');
-      rmBtn.innerHTML = '<i class="fas fa-trash-alt"></i>';
-
-      const mvbtn = document.createElement('span');
-      mvbtn.classList.add('move');
-      mvbtn.innerHTML = '<i class="fas fa-ellipsis-v">';
-
-      p.appendChild(check);
-      p.appendChild(taskDescription);
-
-      listItem.appendChild(p);
-      listItem.appendChild(rmBtn);
-      listItem.appendChild(mvbtn);
-
-      lists.appendChild(listItem);
-
-      // Remove a task
-      rmBtn.addEventListener('click', () => {
-        Todos.removeTask(todo.index);
-        Todos.updateIndex();
-        window.location.reload();
-      });
-
-      // Update todo text
-      taskDescription.addEventListener('click', () => {
+  // Display task on page
+  static displayTask = () => {
+    const todos = Todos.retrieveTask();
+
+    todos.forEach((todo) => {
+      Actions.addNewTask(todo);
+    });
+  }
+
+  // Add and new Task
+  static addNewTask = (todo) => {
+    const listItem = document.createElement('div');
+    listItem.classList.add('list-item');
+
+    const p = document.createElement('p');
+    p.classList.add('desc');
+    p.id = `${todo.index}`;
+
+    const check = document.createElement('input');
+    check.type = 'checkbox';
+    check.classList.add('checkbox');
+    check.setAttribute('aria-label', 'Mark as done');
+    check.checked = todo.completed;
+
+    const taskDescription = document.createElement('span');
+    taskDescription.classList.add('editable');
+    taskDescription.innerText = `${todo.description}`;
+    taskDescription.setAttribute('contenteditable', 'true');
+
+    const rmBtn = document.createElement('button');
+    rmBtn.classList.add('rm-btn');
+    rmBtn.classList.add('hidden');
+    rmBtn.setAttribute('aria-label', 'Delete task');
+    rmBtn.innerHTML = '<i class="fas fa-trash-alt"></i>';
+
+    const mvbtn = document.createElement('span');
+    mvbtn.classList.add('move');
+    mvbtn.innerHTML = '<i class="fas fa-ellipsis-v">';
+
+    p.appendChild(check);
+    p.appendChild(taskDescription);
+
+    listItem.appendChild(p);
+    listItem.appendChild(rmBtn);
+    listItem.appendChild(mvbtn);
+
+    lists.appendChild(listItem);
+
+    // Remove a task
+    rmBtn.addEventListener('click', () => {
+      Todos.removeTask(todo.index);
+      Todos.updateIndex();
+      window.location.reload();
```

Comment:
- [ ] [OPTIONAL] Kindly avoid using the `window.location.reload` because it reloads the page anytime a user make an action which does not give a good user experience